h/auth/util.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 
+from pyramid import authorization
```

Comment:
`IAuthorizationPolicy` is supposed to come from `pyramid.interfaces`, not `pyramid.authorization`.